airbyte-ci/connectors/pipelines/pipelines/bases.py

```diff
@@ -618,5 +628,8 @@ def print(self):
         details_instructions = Text("ℹ️  You can find more details with step executions logs in the saved HTML report.")
         to_render = [step_results_table, details_instructions]
 
+        if self.pipeline_context.dagger_cloud_url:
```

Comment:
Aren't you already logging this on line 481?

Reply:
This is a different Report's `print`. One is ConnectorReport.print and the other is Report.print